src/store/storage-controller.ts

```diff
@@ -111,19 +109,43 @@ export class StorageController {
     return messages;
   }
 
-  public static async delete(
-    messageStore: MessageStore,
-    dataStore: DataStore,
+  public async delete(
     tenant: string,
     message: BaseMessage
   ): Promise<void> {
     const messageCid = await Message.getCid(message);
 
     if (message.descriptor.dataCid !== undefined) {
-      await dataStore.delete(tenant, messageCid, message.descriptor.dataCid);
+      await this.dataStore.delete(tenant, messageCid, message.descriptor.dataCid);
     }
 
-    await messageStore.delete(tenant, messageCid);
+    await this.messageStore.delete(tenant, messageCid);
+  }
+
+  public get MessageStore(): MessageStore { return this.messageStore; }
+
+  public queryMessages(tenant: string, filter: Filter, options?: MessageStoreOptions): Promise<BaseMessage[]> {
+    return this.messageStore.query(tenant, filter, options);
+  }
+
+  public putMessage(tenant: string, messageJson: BaseMessage, indexes, options?): Promise<void> {
+    return this.messageStore.put(tenant, messageJson, indexes, options);
+  }
+
+  public getMessage(tenant: string, messageCid: string): Promise<BaseMessage|undefined> {
+    return this.messageStore.get(tenant, messageCid);
+  }
+
+  public getData(tenant: string, messageCid: string, dataCid: string): Promise<GetResult|undefined> {
+    return this.dataStore.get(tenant, messageCid, dataCid);
+  }
+
+  public appendEvents(tenant: string, messageCid: string): Promise<string>|undefined {
```

Comment:
i think this should just be `appendEvent` since it's only dealing with a single event at a time

Reply:
Addressed in https://github.com/TBD54566975/dwn-sdk-js/pull/338/commits/ecb7e0da3886d0d2023341165c753d09504f9ea2